easy_my_coop_eater/models/coop.py

```diff
@@ -1,25 +1,26 @@
-# -*- coding: utf-8 -*-
-from openerp import api, models
+from odoo import api, models
 
 
 class SubscriptionRequest(models.Model):
-    _inherit = 'subscription.request'
+    _inherit = "subscription.request"
 
     def get_eater_vals(self, partner, share_product_id):
         vals = {}
         eater = share_product_id.eater
 
         if partner.is_company or partner.age < 18:
-            eater = 'eater'
+            eater = "eater"
 
-        vals['eater'] = eater
+        vals["eater"] = eater

```

Comment:
there is a convention about single quote and simple quote ? just for my info

Reply:
not sure. [black formatter](https://github.com/psf/black) changes the simple quote in double quote